AEPCampaignClassic/Sources/CampaignClassic+PublicAPI.swift

```diff
@@ -49,7 +49,7 @@ public extension CampaignClassic {
     ///
     /// - Parameter userInfo :  a dictionary containing `_mId` and `_dId` received in the push message payload, or in the
     ///                             launch options before opening the application
-    static func trackNotificationClick(withUserInfo userInfo: [String: Any]) {
+    static func trackNotificationClick(withUserInfo userInfo: [AnyHashable: Any]) {
```

Comment:
I don't think we should use generic "AnyHashable"  in the public API. We _require_ the String keys "_dId" and "_mId" to be present in the provided dictionary so being specific makes more sense here.

Reply:
The API from which we get the userInfo in swift gives away `[AnyHashable: Any]`
https://developer.apple.com/documentation/usernotifications/unnotificationcontent/1649869-userinfo

Keeping it easier for our customers to provide notification.content.userInfo directly rather than converting.

I have tested this in Objective-C as well, it works great.

Reply:
from the link above: 
```
The keys in this dictionary must be property-list types—that’s, they must be types that can be serialized into the property-list format.
```
i agree with the change to `AnyHashable`